CMakeLists.txt

```diff
@@ -7,6 +7,7 @@ endif()
 
 add_library(kdalgorithms INTERFACE)
 target_include_directories(kdalgorithms INTERFACE src/)
+add_library(KDAB::KDAlgorithms ALIAS kdalgorithms)
```

Comment:
Minor: Would move this next to the initial `add_library()` call but that's just cosmetics.

Reply:
OK, will do.